store/src/java/com/zimbra/cs/html/DefangFactory.java

```diff
@@ -45,30 +46,29 @@
     private static NoopDefang noopDefang = new NoopDefang();
 
     /**
-     * This defanger is used for OWASP
+     * The instance of the owasp html defanger
      */
-    private static OwaspHtmlSanitizer owaspDefang = new OwaspHtmlSanitizer();
+    private static OwaspDefang owaspDefang = new OwaspDefang();
     
     /**
      * 
      * @param contentType
      * @return
      */
     public static BrowserDefang getDefanger(String contentType){
```

Comment:
Please document the significance of content type in returning different types of defanger.